### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by karanpreetkaur**
Adding CONTRIBUTING.md 

Note- The link to Code of Conduct will be added once we have that file in master repository.

### CODE REVIEW

**Review by thayeylolu — APPROVED** [review]
looks good to me!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+You can contribute in many ways. This document outlines how to propose a change to the **credit_default_prediction** project. 
+
+# Types of Contributions
+---
+
+### Minor Changes
+Small typos or grammatical errors in documentation may be edited directly using
+the GitHub web interface, as long as the changes are not made in the _source_ file.
+
+*  YES: you edit a roxygen comment in a `.R` file below `R/`.
```
Comment (liannah) [inline CONTRIBUTING.md:14]: do we have such files ?

Comment (Arushi282) [reply CONTRIBUTING.md]: yeah I think since we are working with python files, you can mention ".py" and docstrings instead?

Comment (karanpreetkaur) [reply CONTRIBUTING.md]: @liannah Thanks for highlighting it , I have updated it now. So, basically the idea here is to allow contributors to make docstring or comments changes directly but to restrict editing the source codes without creating issue

Comment (karanpreetkaur) [reply CONTRIBUTING.md]: @Arushi282 Can you do review now ?

Comment (liannah) [reply CONTRIBUTING.md]: Looks good to me, I am approving. We can wait for @Arushi282 's approval as well.

**Review by liannah — APPROVED** [review]
Looks good to me

File: CONTRIBUTING.md (hunk 1 above)
Comment (Arushi282) [reply CONTRIBUTING.md]: Hi @karanpreetkaur looks great, thanks for making the changes, and the contributing file is basically how others who are not in the core group can make changes to the project on github. Will merge this now.

**Pull request merged**